test/shared/box.lua

```diff
@@ -134,6 +134,28 @@ box.once('initialization', function()
     local tuple = yaml.decode(yml)[1]
     tuple[1] = "12345"
     box.space._schema:insert(tuple)
+
+    --gh-151 Support new _index system space format
+    -- After tarantool-1.7.5-153-g1651fc9be the new _index format
+    -- was introduced. When an index part uses is_nullable or
+    -- collation parameter, then the new format will be used.
+    if tarantool_version_at_least(1, 7, 6, 0) then
```

Comment:
I'm not sure that a space information will be fetched if there are no requests to it. The way to update a schema that is implemented here is not our usual so called 'soft automatic schema reload' (see #62). Can you verify, please?

NB: Need to enable 2.2 in testing.

Reply:
If I understand you correctly - it's fetched.

Reply:
> I'm not sure that a space information will be fetched if there are no requests to it. The way to update a schema that is implemented here is not our usual so called 'soft automatic schema reload' (see #62). Can you verify, please?

Will be checked as in https://github.com/tarantool/tarantool-php/issues/127

Reply:
> NB: Need to enable 2.2 in testing.

Enabled.